appveyor.yml

```diff
@@ -16,11 +16,13 @@ install:
   - ps: dotnet tool install -g GitVersion.Tool
   - ps: dotnet tool install -g dotnet-format
   - ps: dotnet tool install -g dotnet-sonarscanner -v:m
+  - ps: dotnet tool install -g coverlet.console --version 1.6.0
 
 before_build:
   - nuget restore
   - ps: dotnet format --check --dry-run
   - ps: dotnet gitversion /l console /output buildserver
+  - ps: coverlet Ductus.FluentDocker.Tests/bin/Debug/netcoreapp3.0/Ductus.FluentDocker.Tests.dll --target "dotnet" --targetargs "test Ductus.FluentDocker.Tests/Ductus.FluentDocker.Tests.csproj --no-build" --format opencover -o ./output/
```

Comment:
There is a better way to integrate this by adding the proper settings in the csproj :)

Reply:
https://github.com/mariotoffia/FluentDocker/blob/master/Ductus.FluentDocker.Tests/Ductus.FluentDocker.Tests.csproj

should depend on coverlet.collector and add runsettings file to change format to opencover :)

Reply:
https://github.com/Microsoft/vstest/issues/605#issuecomment-546674411